src/localauthentication.cs

```diff
@@ -23,6 +23,8 @@ public enum LABiometryType : long {
 		[Obsolete ("Use 'FaceId' instead.")]
 		TypeFaceId = FaceId,
 #endif
+		[iOS (17, 0), Mac (14, 0), MacCatalyst (17, 0)]
+		OpticId = 1L << 2,
```

Comment:
no flags attr for the enum, so should it just be 4?
```suggestion
		OpticId = 4,
```

Reply:
Which is exactly what that value is https://stackoverflow.com/questions/15278383/whats-value-of-11-how-to-calculate-it